src/probe/probe.c

```diff
@@ -800,6 +800,9 @@ static void probe_cb_produce(void *arg, enum notify_id type, void *data)
 			if (ret < 0)
 				goto err;
 		}
+		/* check if more than 75% of buffer size is already used */
+		if (_probe->ext_dma.dmapb.avail > 0.75 * _probe->ext_dma.dmapb.size)
```

Comment:
We should be able to pass thsi in via the command line, alongside the buffer size (otherwise we use defaults).

Reply:
Are you sure this is FW change? Looks like you was thinking about probe app

Reply:
Your right, the floating point made me think it was the app ! There should be no floating point here, but can do
```
if (avail > size >> 2)
```

Reply:
ups, sorry then, updated with your approach